components/lib/crash/src/main/java/mozilla/components/lib/crash/notification/CrashNotification.kt

```diff
@@ -15,55 +15,51 @@ import mozilla.components.lib.crash.Crash
 import mozilla.components.lib.crash.CrashReporter
 import mozilla.components.lib.crash.R
 import mozilla.components.lib.crash.prompt.CrashPrompt
+import mozilla.components.lib.crash.service.SendCrashReportService
 import mozilla.components.support.base.ids.notify
 
-private const val NOTIFICATION_CHANNEL_ID = "Crashes"
-private const val NOTIFICATION_TAG = "mozac.lib.crash.CRASH"
 private const val NOTIFICATION_SDK_LEVEL = 29 // On Android Q+ we show a notification instead of a prompt
 
+internal const val NOTIFICATION_CHANNEL_ID = "Crashes"
+internal const val NOTIFICATION_TAG = "mozac.lib.crash.CRASH"
+
 internal class CrashNotification(
     private val context: Context,
     private val crash: Crash,
     private val configuration: CrashReporter.PromptConfiguration
 ) {
     fun show() {
         val pendingIntent = PendingIntent.getActivity(
-            context, 0, CrashPrompt.createIntent(context, crash), 0
+                context, 0, CrashPrompt.createIntent(context, crash), 0
```

Comment:
No need to change that in the commit. But this indentation seems to come from the default AS formatting that is a bit different than what ktlint does.

You can get Android Studio to follow ktlint (on Mac) like this:
```
brew install ktlint
ktlint --apply-to-idea-project --android
```

Reply:
After applying and restarting android studio.  It still wants to do double intent for this line.  I'll look into why.  Thanks for the ktlint apply commands.